src/response/generate_api_token_response.rs

```diff
@@ -0,0 +1,17 @@
+use std::time::SystemTime;
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct ApiKeyWithEndpoint {
+    pub api_key: String,
+    pub endpoint: String,
+}
+
+/// The result of a generate api token operation.
+#[derive(Debug)]
+pub struct MomentoGenerateApiTokenResult {
+    pub api_token: String,
```

Comment:
same here about deriving debug

Reply:
Good point, probably undesirable to print any secret material as a side-effect of a well-intentioned log::info. I'd recommend not printing refresh token (or printing `refresh_token: "<redacted>"` to help keep it friendly and clear where it lives)

Reply:
fixed in https://github.com/momentohq/client-sdk-rust/pull/132/commits/a10baf2df4d874788a9eae80e49a54fbedf939bd.